books/RayTracingInOneWeekend.html

```diff
@@ -1429,8 +1430,56 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [rtweekend-initial]: <kbd>[rtweekend.h]</kbd> The rtweekend.h common header]
 
+For all main program files, we will include `rtweekend.h` first, so most other header files (where
+the bulk of our code will reside) can assume that they are free to assume that these definitions are
+already available. (Headers included inside `rtweekend.h` still need to include any of their
+dependencies.) Because of this, we'll make some updates with this assumption in mind.
```

Comment:
I would drop the "Because of this, ".

Reply:
Done. Man, I really had a mouth full of marbles on this one.